src/views/homepage/components/featured-content-grid/certifications-featured-card.tsx

```diff
@@ -23,13 +23,8 @@ const CertificationsFeaturedCard = () => {
 					width={373}
 				/>
 				<div className={s.certificationsCardContentText}>
-					<Heading level={2} size={400} weight="bold">
-						{title}
-					</Heading>
-					<Text asElement="p" size={300} weight="regular">
-						Earn certifications to verify your skills and communicate your
-						proficiency with HashiCorp multi-cloud products.
-					</Text>
+					<h2 className={s.title}>{title}</h2>
+					<p className={s.body}>{body}</p>
```

Comment:
🗒️ I took out `Heading` and `Text` since the styles those apply (`font-size` in particular) change based on `@media` queries.